src/test/java/com/puppycrawl/tools/checkstyle/grammar/AstRegressionTest.java

```diff
@@ -255,6 +256,13 @@ public void testInputJava14TextBlocksTabSize() throws Exception {
             getNonCompilablePath("java14/InputJava14TextBlocksTabSize.java"));
     }
 
+    @Issue("https://github.com/checkstyle/checkstyle/issues/8651")
```

Comment:
I do not think this annotation is good.
we have git for this, and while we are strict on format for commit messages it is better to keep such information in git (hidden by default). Reader of code should focus on code first, any distractions(extra code lines) should be avoided.

Does it make sense to forbid it by Checkstyle validation ? as illegal class
the same as we forbid `@Test(expected = SomeException.class)` annotation of junit , will be enforce at https://github.com/checkstyle/checkstyle/issues/9137

Reply:
https://github.com/checkstyle/checkstyle/issues/9138